### DISCUSSION TITLE
Support github enterprise authentication

### DISCUSSION THREAD

**Discussion by amazingguni** [outline/outline#8046] [Ideas]
Our company uses GitHub Enterprise. 
But i found there is no support for GitHub Enterprise, only public github.
So i'd like to expand outline github feature to use GitHub Enterprise for authentication.

- https://github.com/outline/outline/blob/d8fbe35455fb36582befba34fb4ae3b567d886b3/plugins/github/server/github.ts#L83

I’ll work it and create a pull request. so I’d appreciate your feedback. 😄

**Comment by tommoor** [comment]
Ah yes, we only support GitHub cloud right now. Supporting enterprise is interesting (although more complex), but I think we'd want to restrict that to Outline's licensed edition – if you're paying for GitHub Enterprise that's a company that should be paying for Outline as well.

**Reply by amazingguni** [reply]
i see. thanks for answer

**Comment by djhons** [comment]
I created a free GitHub organization. How should I configure it to allow users in this organization to log in to Outline? I have enabled OAuth, and I need to make my email public to log in properly, but other users who have made their email public are unable to log in.
